lib/git_worktree.rb

```diff
@@ -139,7 +160,7 @@ def mv_dir(old_dir, new_dir)
 
   def fetch_and_merge
     fetch
-    commit = @repo.ref("refs/remotes/#{@remote_name}/master").target
+    commit = @repo.ref("refs/remotes/#{@remote_name}/#{@current_branch}").target
```

Comment:
@Fryguy 
We need to get a list of branches and tags in a repository and then allow the user to switch to a branch/tag and import the Automate Model from that branch/tag.